templates/src/framework/index.js

```diff
@@ -26,7 +26,9 @@ export default function() {
   const render = Component => {
     ReactDOM.render(
       <Provider store={store}>
-        <ConnectedRouter history={history}>{Component}</ConnectedRouter>
+        <ConnectedRouter history={history} store={store}>
```

Comment:
`Provider` of `react-redux` version 5.x passed `store` down to the `ConnectedRouter` using context, so it works without `store` as a `props`. On version 6.x, `store` should be added in `ConnectedRouter` because there was a breaking change.